controlpanel/api/cluster.py

```diff
@@ -329,3 +330,20 @@ def delete_parameter(name):
         if not is_ignored_exception(e):
             raise e
 
+
+def get_repositories(user):
+    repos = []
+    github = Github(user.github_api_token)
+    for name in settings.GITHUB_ORGS:
+        org = github.get_organization(name)
+        repos.extend(org.get_repos())
+    return list(repos)
```

Comment:
isn't repos already a `list`?

Reply:
Oops, a leftover from when I was using `set`s to get rid of dupes. There can't be any anyway.